### PR TITLE
feat: added landing page and docs page

### PR DESCRIPTION
**Pull request by surya-nemiwal**
created a landing page for component library
Screenshot (74)

created a documentation page for component library
Screenshot (75)

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
please add multiple commits with small number of lines of code in a PR with large amt of code.
Good work

File: Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    padding: 1rem;
+    margin: 0.5rem;
+}
+
+h2{
+    margin: 0rem 1rem;
+}
+
+.alert-primary{
+    background-color: rgb(222, 184, 135);
```
Comment (chanchal16) [inline Alert/alert.css:18]: Use variables for colors

File: Card/card.css [hunk 2]
```diff
@@ -0,0 +1,306 @@
+body {
+  font-family: "Quicksand", sans-serif;
```
Comment (chanchal16) [inline Card/card.css]: As you are using this font at multiple places you store it in variable.

**Review by chanchal16 — COMMENTED** [review]
The UI looks great! Good work👍

File: Alert/alert.css (hunk 1 above)
Comment (surya-nemiwal) [reply Alert/alert.css:18]: these colors are used only once so that is why I didn't made variables for these colors

File: Alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
+body{
+    font-family: 'Quicksand', sans-serif;
+    font-size: 20px;
```
Comment (Sharmaryan) [inline Alert/alert.css:3]: you can use rem, as you have used it everywhere in your code

File: Avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+h2{
+    margin: 0rem 1rem;
+}
+
+.avatar-container ul{
+    text-align: center;
+    display: block;
+    padding: 0rem;
```
Comment (Sharmaryan) [inline Avatar/avatar.css:21]: write only 0 and remove rem

File: Card/card.css (hunk 2 above)
Comment (surya-nemiwal) [reply Card/card.css]: changes done

File: Badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+.badge-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  border-radius: 1rem;
+  box-shadow: inset 0rem 0rem 0.5rem gray;
+  padding: 1rem;
+  margin: 0.5rem;
```
Comment (Sharmaryan) [inline Badge/badge.css:14]: for values less than 1rem use px

File: Badge/badge.css [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  top: -0.2rem;
+  right: -0.3rem;
+}
+
+.icon-with-badge span {
+  font-size: 2.3rem;
+}
+
+.badge {
+  color: white;
```
Comment (Sharmaryan) [inline Badge/badge.css:61]: can make use of variables

**Review by Sharmaryan — APPROVED** [review]

**Pull request merged**